### DISCUSSION TITLE
Wildcards Stopped loading

### DISCUSSION THREAD

**Discussion by Lola-Ice** [lllyasviel/Fooocus#3656] [Q&A]
I use wildcards for batch rendering.

Yesterday Fooocus mysteriously stopped recognizing any wildcards. It says in the console that no such file xxxx.txt exists and treats the wildcard title as a normal word.

I have tried several times to change the title and save as .txt with the same result. I reduced the number of wildcards as I had quite a few. It did not help.

I tried also to restart the program. I restarted the system. Still the same message

It worked perfectly fine until yesterday.

Fooocus 2.5.5 Windows 10

**Comment by Rogala** [comment]
Make sure that the file is encoded with utf-8
